Asterix/optics/coronagraph.py

```diff
@@ -204,16 +203,10 @@ def EF_through(self,
                         Can also be a float scalar in which case entrance_EF is constant
                         default is 1.
                         electrical field created by the downstream aberrations introduced directly in the Lyot Stop
-
-        save_all_planes_to_fits: Bool, default False.
-                if True, save all planes to fits for debugging purposes to dir_save_all_planes
-                This can generate a lot of fits especially if in a loop so the code force you
-                to define a repository.
         
         dir_save_all_planes : default None. 
-                                directory to save all plane in
-                              fits if save_all_planes_to_fits = True
-
+                               if not None, directory to save all planes in fits for debugging purposes.
+                               This can generate a lot of fits especially if in a loop, use with caution
```

Comment:
Pleas unindent this as much as acceptable, it just keeps creeping inwards for no reason.